mix.exs

```diff
@@ -76,6 +76,7 @@ defmodule Protobuf.Mixfile do
       files: ~w(
         mix.exs
         README.md
+        lib/elixirpb
```

Comment:
I think we can remove the `:files` config all together. It already seems to include what hex includes by default https://github.com/hexpm/hex/blob/main/lib/hex/package.ex#L4.

Reply:
@ericmj Thanks, that sounds reasonable. I've updated my patch to remove the `:files` config entirely.